### ISSUE TITLE
There is a little bug in the tap switch button assignment.

### ISSUE THREAD

**Issue by JFg111** [Hyrules/WinHue#260]
There is a little bug in the tap buttons assignment.
Buttons 4 and 3 are inverted meaning that assigning a scene to button 4 on the tap switch sensor icon does assign the scene for button 3 on the real tap switch, same for button 3 which is assigned to button 4 on the real switch.
I don't know if it does that with other tap switch, on mine that is the result I got in assigning button.
Otherwise, everything is working fine.

**Comment by Hyrules** [comment]
Yep you are right those 2 where reverse. I have corrected this in the code. Thanks for reporting it. If you want to manually bypass this you can create the rule instead of using the HueTap Creator.
